L13-Python-Interceptors.md

```diff
@@ -0,0 +1,285 @@
+gRPC Python Interceptors
+----
+* Author(s): lidizheng
+* Approver: ericgribkoff
+* Status: In Review
+* Implemented in: Python
+* Last updated: January 17, 2019
+* Discussion at: https://groups.google.com/d/msg/grpc-io/7qIzyCPWbUA/1NKGXAQHCgAJ
+
+## Abstract
+
+This proposal is a hindsight design document that explains the design and implementation details of the interceptor functionality for gRPC Python, which merged in 2017. This gRFC serves as an effort to stabilize the interceptor APIs.
```

Comment:
"hindsight" -> "retroactive"

Reply:
"which merged in" -> "which was merged in"

Reply:
Thanks, that's more accurate!